src/framework-ondata.js

```diff
@@ -63,7 +63,7 @@ module.exports = function(payload, socketIndex) {
 
              new Promise(function( resolve ) { 
                  // Execute fRunUpdater which will also execute the updater
-                 resolve( wrapper.fRunUpdater( vni ) );
+                 resolve( wrapper.fRunUpdater( vni, payload ) );
```

Comment:
Is payload being used by one of the wrapper's fRunUpdaters?  I didn't see it in either the split wrapper or js wrapper.